industrial_ci/ci_main.sh

```diff
@@ -52,7 +52,7 @@ export HIT_ENDOFSCRIPT=false
 
 source ${ICI_PKG_PATH}/util.sh
 
-trap error ERR
+trap error EXIT
```

Comment:
Do we want to create a new function `exit`? Isn't trapping `EXIT` with `error` function misleading?